components/sharing-dialog/src/__tests__/SharingDialog.test.js

```diff
@@ -49,16 +53,17 @@ describe('SharingDialog widget', () => {
     })
 
     it('renders a Close button', () => {
-        const button = getSharingDialogWidget(props).find(Button)
+        const buttons = getSharingDialogWidget(props).find(Button)
+        const closeButtons = buttons.filterWhere(button => button.html().includes('Close'))
 
-        expect(button).toHaveLength(1)
-        expect(button.html()).toMatch('Close')
+        expect(closeButtons).toHaveLength(1)
```

Comment:
Since we're mounting it finds more components than it did with shallow, so I have to do some extra filtering here and in the test below.